crates/context_aware_config/src/api/config/handlers.rs

```diff
@@ -98,52 +99,38 @@ fn is_not_modified(max_created_at: Option<NaiveDateTime>, req: &HttpRequest) ->
     max_created_at.is_some() && parsed_max <= last_modified
 }
 
-async fn generate_cac(
+pub fn generate_config_from_version(
+    version: Option<Value>,
```

Comment:
```suggestion
    version: Option<i64>,
```

Why to parse from a `Value` here in this function, give a `i64` value

Reply:
@ShubhranshuSanjeev this function is getting called at three places , parsing query params there into i64 would be redundant, thats why parsing here at once